src/blurredLocation.js

```diff
@@ -8,13 +8,21 @@ BlurredLocation = function BlurredLocation(options) {
   options.pixels = options.pixels || 400;
 
   options.gridSystem = options.gridSystem || require('./core/gridSystem.js');
+  options.Interface = options.Interface || require('./ui/Interface.js');
 
   gridSystemOptions = options.gridSystemOptions || {};
   gridSystemOptions.map = options.map;
   gridSystemOptions.gridWidthInPixels = gridWidthInPixels;
   gridSystemOptions.getMinimumGridWidth = getMinimumGridWidth;
   gridSystem = options.gridSystem(gridSystemOptions);
 
+  InterfaceOptions = options.InterfaceOptions || {};
+  InterfaceOptions.Id = {latId: 'lat', lngId: 'lng'}
```

Comment:
Do you need to access from in there? You could in theory pass in `blurredLocation` as an option to `Interface` -- that way it gets access to its parent's methods.